.gitignore

```diff
@@ -88,4 +93,7 @@ ENV/
 .ropeproject
 
 .idea
-sanic/*
\ No newline at end of file
+sanic/*
+
+# mypy
+.mypy_cache/
```

Comment:
Interesting. The rest of the entries must be fairly new, because in my recently created Github Python projects, only this entry exists by default.
I can add the rest of them of course.